airflow-core/docs/howto/deadline-alerts.rst

```diff
@@ -0,0 +1,255 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Deadline Alerts
+===============
+
+The :class:`~airflow.sdk.definitions.deadline.DeadlineAlert` feature is the next evolution of
+the old SLA.  Deadline Alerts allow you to set time thresholds for your DAG runs and automatically
```

Comment:
I think it has been brought up we should not compare Airflow version in docs, I am just wondering if mentioning "the old SLA" goes against that.

Reply:
Currently, the [Tasks docs](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#slas) metion:

```The SLA feature from Airflow 2 has been removed in 3.0 and will be replaced with a new implementation in Airflow 3.1```

So we could update the above to something like 

```The SLA feature from Airflow 2 has been removed in 3.0 and was replaced in Airflow 3.1 with Deadlines Alerts (+link)```

And delete the comparison in this section (```The :class:`~airflow.sdk.definitions.deadline.DeadlineAlert` feature is the next evolution of
the old SLA.```).

What do you think?

Reply:
I think it's a good solution.  I'll make it happen.  If someone has a better solution, we can always change it again later.